src/node/rpc/member_frontend.h

```diff
@@ -1241,11 +1241,13 @@ namespace ccf
           submitted_shares_count = share_manager.submit_recovery_share(
             args.tx, args.caller_id, raw_recovery_share);
         }
-        catch (const std::logic_error& e)
+        catch (const std::exception& e)
         {
+          auto error_msg =
```

Comment:
Can you please change this to follow the pattern we are using for other errors we catch
have an error log that prints somethign went wrong, followed by a debug log that prints e.what()
This ensures if there is something confidentual in e.what() we do not leak it

Reply:
Very good point! I was definitely useful for debugging but should not be included in default logging.